csi/csi_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+CSI Interface for OSD
+Copyright 2017 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package csi
+
+import (
+	"net"
+	"reflect"
+	"testing"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+// csiTestService is a simple struct used abstract
+// the creation and setup of the gRPC CSI service
+type csiTestService struct {
```

Comment:
Same comment as above. This is within the csi package, so doesn't really need a qualifier for the struct names. If this was public ie CsiTestService, "go lint" would have complained.

Reply:
Cool, since this one is not exported, I'll change the name.